packages/realm/src/Realm.ts

```diff
@@ -478,6 +632,15 @@ export class Realm {
     }
   }
 
+  /**
+   * Searches for a Realm object by its primary key.
+   * @param type - The type of Realm object to search for.
+   * @param primaryKey - The primary key value of the object to search for.
+   * @throws If type passed into this method is invalid or if the object type did
```

Comment:
I'm wondering if it make sense to pick an order of these annotations? So `@return` is always before `@throw` and perhaps we could add an eslint plugin for that? Just lower the possible variations among these comments.

Reply:
The way old docs were made, return was always after the throw, so I kept it this way, generally I kept this order:
param > throws > return > since